### ISSUE TITLE
Experiment with Lambda

### ISSUE THREAD

**Issue by glassresistor** [codeforamerica/intake#1030]

**Comment by glassresistor** [comment]
On Monday I was able to successfully deploy to Lambda using Zappa with minimal code changes.

It took about 5hrs today deploying a CMR development site to AWS lambda and 3 were spent debugging issues unrelated to Zappa.  It working with our current code base without any changes at all, except adding a new settings file.  I had a handful of small issues that have all been solved.  Currentlycelery tasks to run during the request  but converting them to Lambda events should be easy. 

I've confirmed that pdfparser.jar works on lambda and was able to submit an application and run FillablePdf.fill_submission as a management command to confirm.  I was also able to log into the admin and upload a pdf so file uploads are working as well.
This is all very much so a good sign.

I've been worried that since 1 request = 1 instance, that 1 request would also equal 1 database connection.  This would slow down each request bu having to connect to db first and also cause problems if we had say 100 simultaneous requests.  on a normal server 100 requests would run 2 or 4 at a time sharing 2 or 4 database connections.
so a single server means slow response times but it also means that  you never hit the 100 connection limit that is default for postgres.

The easiest way to test this is to try and hit a url as fast as possible.  I did this locally and requested 150pages at the same time and then later 250  using firefox.

The good news is that I could do hundreds of requests at the same time and my response time stayed ~500ms.  I was also not able to cause an error but I did see the number of database connections jump to 50 and then step down.  I believe this means if someone DDOSed our website they could easily trigger db connection errors site wide.

For reference I can take down the staging server with about 20 simultaneous requests.  In this case what happens is requests start timing out.

This isn't a problem we should worry about much but I will be looking into connection pooling and limiting our instances to <100
